DogsList/.swiftlint.yml

```diff
@@ -7,6 +7,9 @@ disabled_rules: # rule identifiers turned on by default to exclude from running
   - trailing_whitespace
   - identifier_name
   - type_name
+  - file_length
```

Comment:
do not disable rules, use default settings and fix you code

Reply:
Fixed.